client/src/main/proto/spine/base/command.proto

```diff
@@ -26,6 +26,8 @@ option java_outer_classname = "CommandProto";
 option java_multiple_files = true;
 option java_generate_equals_and_hash = true;
 
+option (type_url_prefix) = "type.spine3.org";
```

Comment:
Please put this option immediately before the `java_package` option. No blank line is needed in between.